tests/integration-tests/tests/efa/test_efa.py

```diff
@@ -77,3 +78,21 @@ def _test_osu_benchmarks_openmpi(remote_command_executor, scheduler_commands, te
     output = remote_command_executor.run_remote_command("cat /shared/osu.out").stdout
     latency = re.search(r"0\s+(\d\d)\.", output).group(1)
     assert_that(int(latency)).is_less_than(20)
+
+
+def _test_osu_benchmarks_intelmpi(remote_command_executor, scheduler_commands, test_datadir, slots_per_instance):
```

Comment:
since this code is exactly the same as the previous function can we create a third parametrized function (_run_osu_benchmarks) that accepts as parameter the mpi library to use and then _test_osu_benchmarks_openmpi and _test_osu_benchmarks_intelmpi will simply call that function?

Reply:
Updated!